default.nix

```diff
@@ -604,9 +605,9 @@ let overrideCabal = pkg: f: if pkg == null then null else lib.overrideCabal pkg
 in let this = rec {
   overrideForGhcjs = haskellPackages: haskellPackages.override {
     overrides = self: super: {
-      mkDerivation = drv: super.mkDerivation.override {
-        hscolour = ghc.hscolour;
-      } (drv // {
+      hscolour = ghc.hscolour;
```

Comment:
`mkDerivation` no longer has an `override` attribute. However, it uses the `hscolour` package as it exists in `self.hscolour` so we can override it there.